example/src/main/java/org/secondbase/example/main/HelloSecondBase.java

```diff
@@ -15,8 +15,12 @@ private void printVariable() {
     }
 
     public static void main(final String[] args) throws SecondBaseException {
+        final String[] realArgs = new String[] {
+                "--service", "test",
+                "--datacenter", "dc1"
+        };
         final HelloSecondBase helloSecondBase = new HelloSecondBase();
-        new SecondBase(args, new Flags().loadOpts(helloSecondBase));
+        new SecondBase(realArgs, new Flags().loadOpts(helloSecondBase));
```

Comment:
This is just confusing. This class loads a flag called "var", which is never used. Instead you use "service" and "datacenter", which isn't defined anywhere? This will just crash, right?